index.php

```diff
@@ -58,9 +58,11 @@ function rss2html($url)
 						}
 					}
 
-					echo rss2html("http://blog.openrailwaymap.org/".$lang.".rss");
+					echo rss2html("https://blog.openrailwaymap.org/".$lang.".rss");
 				?>
 			</div>
+
+			<p><a href="https://www.openrailwaymap.org/imprint"><?php echo ($lang == "en") ? "Imprint & Privacy Policy" : "Impressum & Datenschutz" ?></a></p>
```

Comment:
404

And the check should be the other way round: the German version for de, the English version otherwise

Reply:
You are right, the check should be the other way round. I fixed it.

Currently the link returns a 404 because the mod_rewrite redirection is introduced in https://github.com/OpenRailwayMap/OpenRailwayMap/commit/f73c0eba0ecdaee222fb6dbf7b2178f36bc277b1. My plan is:
1. merge this PR
2. update submodule blog in branch gdpr in https://github.com/OpenRailwayMap/OpenRailwayMap
3. create pull request for branch gdpr in https://github.com/OpenRailwayMap/OpenRailwayMap